### PR TITLE
Add support for clojure

### PR DESCRIPTION
**Pull request by abhi18av**
Thanks for the great project!

This PR proposes to add support for `clojure`.

https://github.com/protojure/protoc-plugin

### CODE REVIEW

**Comment by abhi18av** [comment]
Hello @ido-namely ,

Would it be possible for anyone from the team to take a look at this PR :) 

I'm looking forward to using this in a downstream project of mine.

**Comment by abhi18av** [comment]
Just a quick update, I've resolve the merge conflicts.

**Comment by abhi18av** [comment]
Hello @ido-namely , not sure if something pending for this PR. Please let me know 👍

**Comment by jcburley** [comment]
@abhi18av et al – please refresh this PR and I'll look at it anew. Thanks!

**Comment by abhi18av** [comment]
Hi @jcburley , 

I've refreshed the PR - thanks for taking a look into it :)

**Comment by jcburley** [comment]
@abhi18av – it LGTM, but I forgot to ask for tests to be added!

Also, please review the Codacy complaint about `CLOJURE_PROTOC_PLUGIN_VERSION` being unused – I'm not sure what that's about offhand.

Will see whether the current CI passes; I kicked it off only a few moments ago.

File: Dockerfile [hunk 1]
```diff
@@ -181,7 +190,9 @@ COPY --from=build /tmp/grpc_web_plugin /usr/local/bin/grpc_web_plugin
 
 COPY --from=build /tmp/protoc-gen-scala /usr/local/bin/
 
-COPY --from=build /go/pkg/mod/github.com/grpc-ecosystem/grpc-gateway/v2@${grpc_gateway_version}/protoc-gen-openapiv2/options /opt/include/protoc-gen-openapiv2/options/
+COPY --from=build /tmp/protoc-gen-clojure /usr/local/bin/
+
+COPY --from=build /go/pkg/mod/github.com/grpc-ecosystem/grpc-gateway/v2@v${grpc_gateway_version}/protoc-gen-openapiv2/options /opt/include/protoc-gen-openapiv2/options/
```
Comment (jcburley) [inline Dockerfile:195]: I think `v2@v$` needs to be `v2@$` (remove the `v` prefix)? Seems to be failing looking up `vv...` as a version identifier.

**Review by jcburley — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 1 above)
Comment (abhi18av) [reply Dockerfile]: Whoops, that's a good catch - I've fixed it now and I think it's ready to trigger another build perhaps

**Pull request closed without merging**